hearpreprocess/util/audio.py

```diff
@@ -13,85 +13,90 @@
 from tqdm import tqdm
 
 
-def mono_wav_and_fix_duration(in_file: str, out_file: str, duration: float) -> None:
+def mono_wav(in_file: str, out_file: str) -> None:
+    """converts the audio to wav format with mono stream"""
+    try:
+        _ = (
+            ffmpeg.input(in_file)
+            .audio.output(out_file, f="wav", acodec="pcm_f32le", ac=1)
+            .overwrite_output()
```

Comment:
Please remove ALL instances of `overwrite_output` in this file.

Please
```
assert not os.path.exists(out_file)
```
for every ffmpeg operation in this file.

The file already being there should be an error.